homeassistant/components/rituals_perfume_genie/manifest.json

```
@@ -0,0 +1,16 @@
+{
+  "domain": "rituals_perfume_genie",
+  "name": "Rituals Perfume Genie",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/rituals_perfume_genie",
+  "requirements": [
+    "pyrituals==0.0.1"
+  ],
+  "ssdp": [],
+  "zeroconf": [],
+  "homekit": {},
+  "dependencies": [],
```

```suggestion
```

---

Thanks for your review @bdraco.
This was automatically generated by a [scaffold script](https://github.com/home-assistant/core/blob/dev/script/scaffold/templates/integration/integration/manifest.json). I will delete lines 9, 10 and 11. Thanks 😄.
Can "dependencies" be removed? It is included in [this minimal example](https://developers.home-assistant.io/docs/creating_integration_manifest).
```suggestion
  "dependencies": [],
```

---

Yes it can be removed.  It uses a `.get()`

https://github.com/home-assistant/core/blob/dev/homeassistant/loader.py#L373